### PR TITLE
Convert tests to scalatest

### PR DESCRIPTION
**Pull request by griff-pyrofex**
CORE-47.  I converted my tests to scalatest format.

### CODE REVIEW

File: storage/src/test/scala/coop/rchain/storage/LmdbTests.scala [hunk 1]
```diff
@@ -7,132 +7,54 @@
...
-      lmdb.displayRowsIntInt()
+      lmdb.put(1, 2)
+      val getInt = lmdb.getInts(1)
+      assert(getInt.get(0) == 2)
+
+      lmdb.put(3, 4)
+      lmdb.put(3, 5)
+      val getInts = lmdb.getInts(3)
+      assert(getInts.get(0) == 4)
+      assert(getInts.get(1) == 5)
```
Comment (kirkwood) [inline storage/src/test/scala/coop/rchain/storage/LmdbTests.scala]: Is a conformant implementation of storage required to preserve order?

File: storage/src/test/scala/coop/rchain/storage/UniTests.scala [hunk 2]
```diff
@@ -916,14 +507,17 @@ object UniTests {
   }
 
   def ArraysEqual(a1: Array[String], a2: Array[String]): Boolean = {
```
Comment (kirkwood) [inline storage/src/test/scala/coop/rchain/storage/UniTests.scala]: a.k.a.
```
a1.sameElements(a2)
```

File: storage/src/test/scala/coop/rchain/storage/LmdbTests.scala [hunk 3]
```diff
@@ -355,33 +244,44 @@ object LmdbTests {
...
           val value = (i + 1 + v).toString
-          print(value + ", ")
           lmdb.put(key, value)
         }
-        println()
       }
 
-      lmdb.displayRowsStrStr()
+      for (i <- 0 until numKeys) {
+        val valuesSeen = scala.collection.mutable.Map[String, Boolean]()
```
Comment (pyrocto) [inline storage/src/test/scala/coop/rchain/storage/LmdbTests.scala]: Can probably optimize this code a lot using sets, i.e. create set of valuesSeen, then use set difference with values.

File: storage/src/test/scala/coop/rchain/storage/LmdbTests.scala (hunk 1 above)
Comment (griff-pyrofex) [reply storage/src/test/scala/coop/rchain/storage/LmdbTests.scala]: No, that's not a requirement.  Good point about that test.

File: storage/src/test/scala/coop/rchain/storage/UniTests.scala (hunk 2 above)
Comment (griff-pyrofex) [reply storage/src/test/scala/coop/rchain/storage/UniTests.scala]: scala> val x = Array("1","2")
x: Array[String] = Array(1, 2)

scala> val y = Array("2","1")
y: Array[String] = Array(2, 1)

scala> x == y
res0: Boolean = false

scala> x.sameElements(y)
res1: Boolean = false

scala> ArraysEqual(x,y)
res2: Boolean = true

Comment (kirkwood) [reply storage/src/test/scala/coop/rchain/storage/UniTests.scala]: Yeah, go ahead and ignore this comment, since it's addressed more fully in #77

**Pull request merged**